### DISCUSSION TITLE
slow unpacking initramfs and stop in the end

### DISCUSSION THREAD

**Discussion by madcry** [epoptes/epoptes#87] [Q&A]
Ubuntu 18.04
When try to start clients (Intel-NUC), the loading stop in the end,
First problem is slow initramfs unpacking 
![pic1](https://user-images.githubusercontent.com/36723580/49005039-c3e80d80-f176-11e8-8f88-5152b30ddfec.jpg)
Loading take 10-15 min, like in slow motion.
And stop in the end
![pic2](https://user-images.githubusercontent.com/36723580/49005103-ec700780-f176-11e8-8129-6fea84cbde42.jpg)
This problem is heppens only with Intel-NUC computers, other client start normaly...
Have any idia why?

**Comment by alkisg** [comment]
Hi, Epoptes doesn't have any code that would affect initramfs unpacking or boot speed. So look elsewhere for the issue, it's not Epoptes related (e.g. maybe the problem is in the kernel, in systemd or in ltsp).

For proof, uninstall Epoptes and try again. If you see that it boots slowly without Epoptes and fast with it, then please reopen this issue.
